README.md

```diff
@@ -175,6 +175,53 @@ e.g. (for json format)
 }
 ```
 
+## Generic Application alerting
+Generic Application alerting can be enabled for selected apps.
+
+This will add alerts that will trigger as specified below
+- Memory consumption > 50%
+- Cpu usage > 60%
+- Disk usage > 60%
+- App Crashes > 1
+- Elevated request failures > 0.1
```

Comment:
maybe 10% instead of 0.1?

Reply:
app alert yaml has  value 0.1 before i parameterized.  https://github.com/DFE-Digital/bat-infrastructure/blob/bde92d4444dd92f1d735b16e2171dc37b6fb89de/monitoring/config/alert.rules.tmpl#L70